k8s/helm/yona-server/templates/01_configmap_properties.yaml

```diff
@@ -44,14 +45,15 @@ data:
     yona.ldap.accessUserDn={{ .Values.ldap.user_dn | default "cn=admin,dc=example,dc=local" }}
     yona.ldap.accessUserPassword={{ .Values.ldap.user_password | default "password" }}
     yona.sms.enabled= {{ .Values.sms.enabled | default "false" }}
-    yona.sms.defaultSenderNumber = {{ .Values.sms.default_sender_number | default "+17184169858" }}
+    yona.sms.defaultSenderNumber = {{ .Values.sms.default_sender_number | default "+12312312312" }}
```

Comment:
We need to do something with the multitude of defaults. Let's make this part of that.